### PR TITLE
feat: validator list

### PR DESCRIPTION
**Pull request by songwongtp**
## Describe your changes
Fixes: CFE-313, CFE-314, CFE-319

### CODE REVIEW

File: src/lib/pages/validators/index.tsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  );
+
+  useEffect(() => {
+    if (router.isReady) track(AmpEvent.TO_VALIDATORS);
+  }, [router.isReady]);
+
+  useEffect(() => {
+    setCurrentPage(1);
+    setPageSize(100);
+  }, [isActive, order, isDesc, debouncedSearch, setCurrentPage, setPageSize]);
```
Comment (evilpeach) [inline src/lib/pages/validators/index.tsx:62]: chainId?

File: src/lib/pages/validators/index.tsx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+            onPageChange={setCurrentPage}
+            onPageSizeChange={(e) => {
+              const size = Number(e.target.value);
+              setPageSize(size);
+              setCurrentPage(1);
+            }}
+          />
+        )}
+      </PageContainer>
+    </div>
```
Comment (evilpeach) [inline src/lib/pages/validators/index.tsx:146]: Can we just we use `<>`?

File: src/lib/pages/validators/components/validators-table/ValidatorsTableBody.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+          totalVotingPower={data.metadata.totalVotingPower}
+          minCommissionRate={data.metadata.minCommissionRate}
+          denomToken={denomToken}
+        />
+      ))}
+    </>
+  ) : (
+    <>
+      {data.items.map((validator) => (
+        <>
```
Comment (evilpeach) [inline src/lib/pages/validators/components/validators-table/ValidatorsTableBody.tsx:72]: add key

File: src/lib/pages/validators/index.tsx (hunk 1 above)
Comment (songwongtp) [reply src/lib/pages/validators/index.tsx:62]: chainId is already handled in `usePaginator`

**Review by evilpeach — APPROVED** [review]

**Pull request merged**